### PR TITLE
Visual Transformer compatibility

### PR DESCRIPTION
**Pull request by pmcgrath249**
* changed activation labels to cope with FC, Transformer w/ 1D embedding and 2DConv
* added Unit test

### CODE REVIEW

**Review by mschrimpf — APPROVED** [review]
perfect.

File: model_tools/activations/core.py [hunk 1]
```diff
@@ -192,12 +192,12 @@ def _package_layer(self, layer_activations, layer, stimuli_paths):
         activations, flatten_indices = flatten(layer_activations, return_index=True)  # collapse for single neuroid dim
         assert flatten_indices.shape[1] in [1, 2, 3]
         if flatten_indices.shape[1] == 1:    # FC
-            flatten_coord_names = ['channel']
+            flatten_coord_names = ['channel', 'channel_x', 'channel_y']
```
Comment (mschrimpf) [inline model_tools/activations/core.py:195]: how come we also need `x` and `y` for fully-connected?

Comment (pmcgrath249) [reply model_tools/activations/core.py:195]: I am currently looking into that. But that is why we got so many weird key errors last time. Apparently (in the tests I ran locally with alexnet), first computing activations for a Conv layer and subsequently for an FC layer results in the key issue. It does not really makes sense to me, this was just a way to circumvent that for now.

Comment (pmcgrath249) [reply model_tools/activations/core.py:195]: https://github.com/brain-score/model-tools/blob/4496106fa2267dc9e63c3ba7f1c99fe9ade71d56/model_tools/activations/core.py#L175 
in package, while creating coords, we are implicitly assuming the same structure for all layers (the little layer_assemblies[0]), i.e. channel, channel_x, channel_y if the first layer we test is a conv layer. that is why we need empty channel_x and _y for subsequent FC layers.

Comment (pmcgrath249) [reply model_tools/activations/core.py:195]: We could change walk_coords to also yield an index.

**Pull request merged**